diff -rupN MIDI-Player-GUI/midi.py MIDI-Player-GUI-new/midi.py
--- MIDI-Player-GUI/midi.py     2025-05-01 14:33:33.021566142 +0000
+++ MIDI-Player-GUI-new/midi.py 2025-05-01 14:34:07.762755088 +0000
@@ -11,10 +11,10 @@ import time
 #if ownPanic() is called (e.g. stop pressed), leave register untouched
 
 #Qsynth
-#port = mido.open_output('Midi Through:Midi Through Port-0 14:0')
+port = mido.open_output('Midi Through:Midi Through Port-0 14:0')
 
 #CH345 USB-Midi
-port = mido.open_output('CH345:CH345 MIDI 1 20:0')
+#port = mido.open_output('CH345:CH345 MIDI 1 20:0')
 
 ###
 hwRegChan = 0
